_docs/whats-new/whats-new.md

```diff
@@ -10,6 +10,11 @@ toc: true
 We launched the Codefresh platform in February this year. Built on Argo, the world’s most popular and fastest-growing open source software delivery, Codefresh unlocks the full enterprise potential of Argo Workflows, Argo CD, Argo Events, and Argo Rollouts, providing a control-plane for managing them at scale.
 Since the launch, we have continued to work on and grow Codefresh. 
 
+
```

Comment:
Please remove this; it's not open yet -  will be released only for the MVP, and I am not sure if we will have a dedicated release notes for these. Otherwise, the release notes are now monthly